Neos.Neos/Classes/Domain/Service/UserService.php

```diff
@@ -215,7 +222,7 @@ public function getCurrentUser()
         if ($this->securityContext->canBeInitialized() === true) {
             $account = $this->securityContext->getAccount();
```

Comment:
Not directly related to this change, but this could return a different account. Shouldn't we use `$this->securityContext->getAccountByAuthenticationProviderName()` instead?

Reply:
nice one, actually this could need some love (but outside of this change if possible), but at least the `getUser()` should pass in the authentication provider of the account returned as otherwise the `getUser()` method will fall back to the defaultAuthenticationProviderName.